# goal definition
GraspCommand grasp_command
---
# result definition
std_msgs/Bool pregrasped
std_msgs/Bool grasped
std_msgs/Bool postgrasped
---
# feedback definition
sensor_msgs/JointState current_joint_state
